konig-spreadsheet/src/main/resources/WorkbookLoader/BigQueryCsvBucket.ttl

```diff
@@ -28,7 +28,11 @@
 	gcp:name "${gcpBucketName}" ;
 	gcp:projectId "${gcpProjectId}" ;
 	gcp:storageClass "${gcpBucketStorageClass}" ;
-	gcp:location   "${gcpBucketLocation}" 
+	gcp:location   "${gcpBucketLocation}" ;	
+	gcp:notificationInfo [
+		gcp:topic "${topicName}" ;
+		gcp:eventTypes "${eventType}";
+	] 

```

Comment:
Per the acceptance criteria, you need to add business logic to the template... 
1. The `notificationInfo` should be present only if  `${topicName}` is defined.
2. If `${eventType}` is not defined, then it must default to `OBJECT_FINALIZE`
3. If `${eventType}` is defined but `${topicName}` is not defined, then throw an exception.  For hints on how to throw an exception from a Velocity template, see [stackoverflow](https://stackoverflow.com/questions/15659200/how-to-throw-an-user-defined-exception-from-velocity-template-script-vtl)

Also, please add a unit test for your code.